File: jungol_1740.cpp
#include <stdio.h>

int primeCheck(int a);

int main(){
	int m, n, i;
	int sum, min;
	
	scanf("%d %d", &m, &n);
	sum = 0;
	min = n;
	
	for(i=m; i<=n; i++){
		if(primeCheck(i)){
			sum += i;
			min = min > i ? i : min;
		}
	}
	
	if(sum == 0){
		printf("-1");
	}
	else {
		printf("%d\n%d", sum, min);
	}
	
	return 0;
}

int primeCheck(int a){
	int i;
	if(a == 1){
		return 0;
	}
	for(i=2; i*i<=a; i++){
		if(a%i == 0){
			return 0;
		}
	}
	return 1;
}

File: jungol_1740.py
import math

def primeCheck(a) :
    if a == 1:
        return False
    
    for i in range(2, int(math.sqrt(a))+1):
        if a%i == 0:
            return False
    
    return True


m = int(input(""))
n = int(input(""))
sum = 0
min = n

for i in range(m, n+1):
    if primeCheck(i):
        sum += i
        if min > i :
            min = i

if sum == 0:
    print('-1')
else :
    print('{0}\n{1}'.format(sum, min))